lib/editor.go

```diff
@@ -125,19 +122,29 @@ func GetEditor() *Editor {
 				buffer:  NewBuffer(),
 				scratch: true,
 			},
-			keyInput: make(chan keys.KeyPress, 32),
+			keyInput:    make(chan keys.KeyPress, 32),
+			defaultSet:  new(HasSettings),
+			platformSet: new(HasSettings),
```

Comment:
These should probably still be `defaultSettings` and `platformSettings` for clarity.